### ISSUE TITLE
Empty param output for each entry when no param output is selected

### ISSUE THREAD

**Issue by animaux** [hananils/association_output#8]
Es scheint als ob im `param_pool` Geisterelemente für jeden Eintrag auftauchen obwohl eigentlich kein Feld bei zur Parameterausgabe ausgewählt ist:

```
 <ds-uebersicht>
        <item handle="1">1</item>
        <item handle="1">1</item>
        <item handle="1">1</item>
        <item handle="1">1</item>
        <item handle="1">1</item>
        …
        <item handle="1">1</item>
        <item handle="1">1</item>
        <item handle="1">1</item>
        <item handle="1">1</item>
        <item handle="1">1</item>
    </ds-uebersicht>
```

**Comment by animaux** [comment]
Um das Problem nochmal genauer darzulegen, es werden immer *sämtliche IDs der verknüpften Bereiche* im Parameter-Pool ausgegeben.

**Comment by nilshoerrmann** [comment]
This extension used output parameters internally to fetch the nested content. These parameters should be removed again later which seems to fail.

**Comment by animaux** [comment]
OK thanks, at least it shouldn’t affect performance then?

**Comment by nilshoerrmann** [comment]
No, it shouldn't.

**Comment by nilshoerrmann** [comment]
Alex, kann es sein, dass Du gleichzeitig eine Union Data Source eingesetzt hast?
Das Problem klingt mir ein bisschen nach https://github.com/brendo/uniondatasource/issues/42.

**Comment by animaux** [comment]
Nope. Ich hab das in allen Installationen mit `association_output`.

**Comment by nilshoerrmann** [comment]
Interessant. Ich kann das Problem bisher nicht reproduzieren, hatte aber oben verlinktes Problem mit der Union Data Source.

**Comment by animaux** [comment]
Hmn, evtl. spielt eine andere Extension eine Rolle? Ich probier mal was gleich.

**Comment by nilshoerrmann** [comment]
Problematisch sind alle Erweiterungen, die auch das Ausgabe-XML manipulieren. Da kann einiges schief gehen.

**Comment by animaux** [comment]
Hmn ok. Hab mal alles deaktiviert was geht. Es bleiben noch folgende Erweiterungen aktiv:

+ Association Field
+ Association Output
+ Association UI: Editor
+ Debug Devkit
+ Field: Image Upload
+ Markdown Text Formatter
+ Order Entries
+ Reflection Field

Hmn.

**Comment by nilshoerrmann** [comment]
Und was nun? Immer noch das Problem?
Wenn ja: Welche PHP-Version habt Ihr denn?

**Comment by nilshoerrmann** [comment]
Und gibt es relevante Fehler in den Logs?

**Comment by animaux** [comment]
Bei SoR ist es ja z. B. auch so :)

**Comment by animaux** [comment]
PHP = 5.3.28-nmm3

**Comment by animaux** [comment]
Keine Fehler.

**Comment by nilshoerrmann** [comment]
Bei SoR sehe ich so was nicht im Debugger. Reden wir über die gleiche Sache?!

**Comment by animaux** [comment]
Ja, sorry. Mache grad zu viele Sachen gleichzeitig …

Dachte gerade es ging nur darum, dass überhaupt etwas als DS-Parameter ausgegeben wird obwohl das nicht im DS-Editor ausgewählt wurde.

Aber es ging ja ursprünglich um die leeren Parameter mit '1' wie im Screenshot. Ich kann leider grad nicht mehr nachvollziehen in welcher Installation das Problem war. Ich such mal nach einer Union-DS in unseren Projekten …

**Comment by nilshoerrmann** [comment]
Danke. Bezüglich SoR: Dort werden aber weder leere noch unangeforderte Parameter ausgegeben. Oder siehst Du da was anderes als ich?

**Comment by animaux** [comment]
Ja, stimmt.
